src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -1933,7 +1933,7 @@ public int Compare(Symbol fst, Symbol snd)
 
                             if (best.IsFromSourceModule)
                             {
-                                arg0 = srcSymbol.Locations.First().SourceTree.FilePath;
+                                arg0 = srcSymbol.GetFirstLocation().SourceTree.FilePath;
```

Comment:
this commit should be the mechanical change of Locations.First() to GetFirstLocation()